README.md

```diff
@@ -2,20 +2,52 @@
 
 ## Overview
 
-Bucky is a testing framework that supports Web System Testing Life Cycle.
+Bucky is a testing framework that supports web system testing life cycle.
 
 ## Feature
 
-* Tests run in parallel
-* Re-run function
-* Codes of test suite are of the form YAML
-* Multiple browser support (Chrome)
-* Some Test Categories support
-  * E2E Tests including JavaScript Error Check
-  * LINK Status Tests in web page
-* Test Report by [bucky-management](https://github.com/lifull-dev/bucky-management)
+* Run tests in parallel
+* Re-run tests whice failed last time
+* Support test code in YAML
+* Multiple browser supported (Currently only Chrome is supported)
+* Customizable test categories
+  * [Default] E2E: E2E tests including JavaScript error check
+  * [Default] Linkstatus: http respond check in web page
```

Comment:
I think http status code check is better.

Reply:
agree!